block/production_test.go

```diff
@@ -36,14 +40,15 @@ func TestCreateEmptyBlocksEnableDisable(t *testing.T) {
 	// Init manager with empty blocks feature disabled
 	managerConfigCreatesEmptyBlocks := testutil.GetManagerConfig()
 	managerConfigCreatesEmptyBlocks.BlockTime = blockTime
-	managerConfigCreatesEmptyBlocks.EmptyBlocksMaxTime = 0 * time.Second
+	managerConfigCreatesEmptyBlocks.MaxIdleTime = 0 * time.Second
 	managerWithEmptyBlocks, err := testutil.GetManager(managerConfigCreatesEmptyBlocks, nil, nil, 1, 1, 0, proxyApp, nil)
```

Comment:
the test has everything named the wrong way
the manager which does not create empty blocks in called managerWithEmptyBlocks
it needs to be flipped around

Reply:
nvm